Tella/Scenes/Uwazi/Entity/Utils/Protocol/UwaziEntityParserProtocol.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UwaziEntityParserProtocol.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/13/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+protocol UwaziEntityParserProtocol {
```

Comment:
https://github.com/Horizontal-org/Tella-iOS/pull/111#discussion_r1453366003
I think all the files under `Tella/Scenes/Uwazi/Entity/` are in the project  but they are not used in the project. (duplicated in other places)  Could you check all the files under `Tella/Scenes/Uwazi/Entity/` ?

Reply:
Yes, they are duplicated. The files were supposed to be removed, it's weird because the are not visible on xcode but they are still in the root folder. I just removed then now